charts/gardener/controlplane/charts/application/templates/apiservice-v1alpha1-settings-gardener-cloud.yaml

```diff
@@ -10,10 +10,16 @@ metadata:
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
 spec:
-  insecureSkipTLSVerify: {{ .Values.global.apiserver.insecureSkipTLSVerify }}
-  {{- if not .Values.global.apiserver.insecureSkipTLSVerify }}
+  {{- if or .Values.global.apiserver.insecureSkipTLSVerify .Values.global.apiserver.createNewCa .Values.global.apiserver.caFromSecret.enabled }}
+  insecureSkipTLSVerify: true
+  {{- else }}
+  insecureSkipTLSVerify: false
+  {{- end }}
```

Comment:
`insecureSkipTLSVerify: true` means you can only use this for developing / experimentation. Is this really the idea? Do you see a way to avoid it in your environment? If not, I am starting to doubt the entire idea of this change.

In any case, I would not bundle setting `insecureSkipTLSVerify: true` with `.Values.global.apiserver.caFromSecret.enabled` (or however it's called) here. These 2 are independent from my perspective. If a CA secret is used, the value of `caBundle` could still be extracted from the secret e.g. by a script and passed here.